src/contexts/Balances/Provider.tsx

```diff
@@ -131,6 +135,9 @@ const Provider: React.FC = ({ children }) => {
       setUsdcBalance(
         new BigNumber(balances[7]).dividedBy(new BigNumber(10).pow(6))
       )
+      setBedBalance(
+        new BigNumber(balances[8]).dividedBy(new BigNumber(10).pow(18))
+      )
       setUniswapEthDpiLpBalance(
         new BigNumber(balances[8]).dividedBy(new BigNumber(10).pow(18))
```

Comment:
let's make sure we bump this up in accordance with adding BED before it

Reply:
oh good catch. will fix that now